vnext/Microsoft.ReactNative/Views/ReactRootControl.cpp

```diff
@@ -607,6 +608,53 @@ void ReactRootControl::ReloadViewHost() noexcept {
   }
 }
 
+void ReactRootControl::AttachBackHandlers(XamlView const &rootView) noexcept {
+  auto rootElement(rootView.as<winrt::UIElement>());
+  if (rootElement == nullptr) {
+    assert(false);
+    return;
+  }
+
+  winrt::Windows::UI::Core::SystemNavigationManager::GetForCurrentView().BackRequested(
+      [this](winrt::IInspectable const & /*sender*/, winrt::BackRequestedEventArgs const &args) {
+        args.Handled(OnBackRequested());
+      });
+
+  // Handle keyboard "back" button press
+  winrt::KeyboardAccelerator goBack{};
+  goBack.Key(winrt::VirtualKey::GoBack);
+  goBack.Invoked(
+      [this](winrt::KeyboardAccelerator const & /*sender*/, winrt::KeyboardAcceleratorInvokedEventArgs const &args) {
+        args.Handled(OnBackRequested());
+      });
+  rootElement.KeyboardAccelerators().Append(goBack);
```

Comment:
>Append [](start = 37, length = 6)

It also must be reversible upon unload.
The ReactRootControl could be initialized/uninitialized multiple times.

Reply:
Added a function to remove the keyboard accelerators attached to the XAML root element on uninitialization.